Setup Windows build on AppVeyor

There have been windows binaries released up to version 1.10 but not for the 1.11 release. Is it possible to get an updated windows binary?

EDIT: I just noticed [the simple build instructions](https://github.com/jonls/redshift/blob/master/HACKING.md) and compiled [my own binary](https://dl.dropboxusercontent.com/s/b1j327cohyxzofv/redshift-1.11-windows.zip).

@manuc66 Thanks for looking into this 👍  The build seems to have failed with "configure: error: Your intltool is too old.  You need intltool 0.50 or later.": https://ci.appveyor.com/project/manuc66/redshift

--- a/README.md
+++ b/README.md
@@ -18,14 +18,14 @@ Build status
 ------------
 
 [![Build Status](https://travis-ci.org/jonls/redshift.svg?branch=master)](https://travis-ci.org/jonls/redshift)
-
-Building from source
---------------------
-
-See the file [HACKING](HACKING.md) for more details on building from source.
+[![Build Status](https://ci.appveyor.com/api/projects/status/github/jonls/redshift?branch=master&svg=true)](https://ci.appveyor.com/project/jonls/redshift)
 
 FAQ
 ---
+
+**How do I install Redshift?**
+
+Use the packages provided by your distribution, e.g. for Ubuntu: `apt-get install redshift` or `apt-get install redshift-gtk`. For developers, please see _Building from source_ and _Latest builds from master branch_ below.
 
 **Where can I find my coordinates to put in the configuration file?**
 
@@ -91,7 +91,13 @@ Please go to [the issue tracker](https://github.com/jonls/redshift/issues) and
 check if your issue has already been reported. If not, please open a new issue
 describing you problem.
 
-Donations
----------
+Latest builds from master branch
+--------------------------------
 
-[![Flattr](http://api.flattr.com/button/flattr-badge-large.png)](https://flattr.com/thing/57936/Redshift)
+- [Ubuntu PPA](https://launchpad.net/~dobey/+archive/ubuntu/redshift-daily/+packages) (`sudo add-apt-repository ppa:dobey/redshift-daily`)
+- [Windows x64](https://ci.appveyor.com/api/projects/jonls/redshift/artifacts/redshift.zip?branch=master)
+
+Building from source
+--------------------
+
+See the file [HACKING](HACKING.md) for more details on building from source.

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,0 +1,24 @@
+image:
+- Visual Studio 2015
+
+environment:
+  MSYS_ARCH: x86_64
+
+build:
+  verbosity: detailed
+
+build_script:
+- echo %PATH%
+- set PATH=C:\msys64\usr\bin;C:\mingw-w64\x86_64-6.3.0-posix-seh-rt_v5-rev1\mingw64\bin;%PATH%
+- set CC=gcc
+- set MSYSTEM=MINGW64
+- bash -lc "pwd"
+- bash -lc "cd $APPVEYOR_BUILD_FOLDER && ./bootstrap"
+- bash -lc "cd $APPVEYOR_BUILD_FOLDER && ./configure --disable-drm --disable-randr --disable-vidmode --enable-wingdi --disable-geoclue --disable-gui --disable-ubuntu --host=x86_64-w64-mingw32"
+- bash -lc "cd $APPVEYOR_BUILD_FOLDER && make"
+
+after_build:
+- cmd: >-
+    7z a redshift.zip ./src/redshift.exe COPYING NEWS README.md
+
+    appveyor PushArtifact redshift.zip

